scripts/changelog/gradle_updater.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable functional/immutable-data */
```

Comment:
This are great! Should we had a JIRA task to insert this lint exception to all other scripts?

Reply:
Let's refactor step by step on need what do you think?

Reply:
I wouldn't do it in this PR, but I'd do another PR that address all other scripts in this section (folder `./scripts`). It's really annoying open a scripts and be flooded with many errors ;). What do you think?

Reply:
Even if it is not that common to open these scripts, deal! 👍🏻